### PR TITLE
Workaround binary issue when reading in log files compiled with pgi

### PR DESCRIPTION
**Pull request by AnnikaLau**

### CODE REVIEW

**Comment by AnnikaLau** [comment]
Cannot link to issue https://github.com/C2SM/icon/issues/12 as it in the icon repository.

**Comment by AnnikaLau** [comment]
launch Jenkins

**Comment by mjaehn** [comment]
@AnnikaLau Is this working for you when you read the logfile on Daint?

**Comment by AnnikaLau** [comment]
> @AnnikaLau Is this working for you when you read the logfile on Daint?

It was but not any longer... I'm trying to figure out what the issue is.

**Comment by AnnikaLau** [comment]
> > @AnnikaLau Is this working for you when you read the logfile on Daint?
> 
> It was but not any longer... I'm trying to figure out what the issue is.

It should work now. Just deleted an import variable by accident before pushing.

**Comment by mjaehn** [comment]
Alright thanks! Currently compiling ICON with pgi... will test this tomorrow morning.

File: create_scaling_table_per_exp.py [hunk 1]
```diff
@@ -153,13 +153,21 @@ def grep(string, filename):
...
-                list_line.append(line)
-                list_iline.append(iline)
-                lo_success = True
-
+        file = open(filename, 'r')
+        count = 0
+        while True:
+            try:  # Some lines are read in as binary with the pgi compilation
+                line = file.readline()
+                count = +1
```
Comment (mjaehn) [inline create_scaling_table_per_exp.py:161]: Script is working fine now, but this should be `count += 1`

**Review by mjaehn — CHANGES_REQUESTED** [review]

File: create_scaling_table_per_exp.py (hunk 1 above)
Comment (AnnikaLau) [reply create_scaling_table_per_exp.py]: Thanks, I adapted it!

**Comment by mjaehn** [comment]
Looks good to me now! 
Btw I checked which line is still read in as binary and it appears to be here (2nd line):
```
 \varname{hydro_evapo_wtr_lake} & \varname{hydro_evapo_wtr_lake} & Evapotranspiration from lake water     
 \\                                                                                                      
 \varname{hydro_evapopot_box} & \varname{hydro_evapopot_box} & Potential evaporation from surface 
```

**Pull request merged**